tests/unit/withOnyxTest.js

```diff
@@ -95,6 +97,7 @@ describe('withOnyx', () => {
     });
 
     it('should update withOnyx subscribing to individual key with merged value if mergeCollection is used', () => {
+        // Todo: ViewWithCollections does not expect a `text` prop
```

Comment:
Who is this todo for / are we planning to clean this up? not sure what this comment means.

Reply:
Not sure who are they for,  it's just something that I've spotted that's out of this PR's scope

Reply:
Please create a new issue for them then. We don't leave random `todos` throughout our codebase for others to wonder how or if they should be actioned.

Reply:
Addressed in #79